frontend/src/components/MapView/LeftPanel/layersPanel/RootAccordionItems/index.tsx

```diff
@@ -1,11 +1,16 @@
 import { LayersCategoryType, MenuItemType } from 'config/types';
-import { memo } from 'react';
+import { memo, useMemo } from 'react';
 import useLayers from 'utils/layers-utils';
 import MenuItem from '../MenuItem';
-import { menuList } from '../../utils';
+import { getDynamicMenuList } from '../../utils';
 
 const RootAccordionItems = memo(() => {
   const { adminBoundariesExtent: extent } = useLayers();
+  const { selectedLayers } = useLayers();
+  const menuList = useMemo(
+    () => getDynamicMenuList(selectedLayers),
```

Comment:
The dynamic menu list allows us to use different layers (monthly vs seasonal) to drive the same menu item with a toggle